dev/io.openliberty.microprofile.health.internal.common/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -0,0 +1,22 @@
+###############################################################################
+# Copyright (c) 2025 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License 2.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-2.0/
+# 
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+###############################################################################
+# -------------------------------------------------------------------------------------------------
+#ISMESSAGEFILE FALSE
+#NLS_ENCODING=UNICODE
+#NLS_MESSAGEFORMAT_NONE
+
+mpHealth=MicroProfile Health
+mpHealth.desc=Configuration for MicroProfile Health.
+
+fileUpdateInterval=Interval time in for updating file based health checks.
+fileUpdateInterval.desc=Values specified will set the period interval for which the file based health checks are updated.
```

Comment:
Should we mention that only seconds and milliseconds are supported?

Reply:
fixed w/ https://github.com/OpenLiberty/open-liberty/pull/30811/commits/0a74039c21072c6346d5488d6ec7351594d215f5